cinch/roles/nginx/templates/etc/nginx/conf.d/errors.conf.snippet.j2

```diff
@@ -0,0 +1,11 @@
+# redirect server error pages to the static pages from ESO_error_pages RPM
```

Comment:
In the cinch repository, we have been avoiding the use of the .j2 extension in templates as it breaks most editor highlighting.